backend/src/test/java/com/redhat/cloud/notifications/db/DbCleaner.java

```diff
@@ -73,7 +72,8 @@ public void clean() {
         for (Class<?> entity : ENTITIES) {
             entityManager.createQuery("DELETE FROM " + entity.getSimpleName()).executeUpdate();
         }
-
+        entityManager.createQuery("DELETE FROM Template templ where templ.id not in (select it.theTemplate.id from IntegrationTemplate it where it.templateKind = :default)")
```

Comment:
Global templates such as Google Chat, Teams, Slack or Drawer should not be cleaned on tests